Analysis/Tasks/PWGHF/taskBPlus.cxx

```diff
@@ -65,7 +65,7 @@ struct TaskBplus {
   Filter filterSelectCandidates = (aod::hf_selcandidate_d0::isSelD0 >= d_selectionFlagD0 || aod::hf_selcandidate_d0::isSelD0bar >= d_selectionFlagD0bar);
   Partition<aod::BigTracks> positiveTracks = aod::track::signed1Pt >= 0.f;
 
-  void process(aod::Collision const&, aod::BigTracks const&, soa::Filtered<soa::Join<aod::HfCandProng2, aod::HFSelD0Candidate, aod::Colls>> const& candidates)
+  void process(aod::Collision const&, aod::BigTracks const&, soa::Filtered<soa::Join<aod::HfCandProng2, aod::HFSelD0Candidate>> const& candidates)
```

Comment:
* Please remove also the table and the `AddCollisionId` task.
* Add missing `cfgc`.